show/main.py

```diff
@@ -797,6 +813,102 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
+
+#
+# 'breakout' group ###
+#
+@interfaces.group(invoke_without_command=True)
+@click.pass_context
+def breakout(ctx):
+    """Show interface breakout"""
```

Comment:
This docstring becomes the help message. It should be a little more descriptive. Even if simply, `Show interface breakout mode info`

Reply:
#done